app/models/message.rb

```diff
@@ -49,6 +49,7 @@ class Message < ApplicationRecord
 
   scope :most_recent_last, -> { includes(:user_message_statuses).order(created_at: :asc) }
 
+  before_save :create_checksum
```

Comment:
as [comment](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3713/files#r575104297)